libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ObjectBuilder.kt

```diff
@@ -9,6 +9,10 @@ abstract class ObjectBuilder(override val customScalarAdapters: CustomScalarAdap
 
   var __typename: String by __fields
 
+  infix fun String.to(value:Any?) {
```

Comment:
This overrides the default `to` that creates a `Pair`. I think it's ok because it reads really well and is only available in the context of an `ObjectBuilder` but we can brainstorm other names.

Reply:
Not a strong preference, but I think I'd rather keep the `this["xxx"]` syntax - I find it slightly less "magic" and, having only 1 way to do it is probably better

Reply:
Fair enough 👍 I'll revert to the existing `this["foo"]` syntax